lambdaclass/curse_of_myrra
# Remove TopDown engine from repository

Closes #1112 

## Motivation & Summary of Changes

Since TopDown Engine is an external dependency we removed it from the repository.

## How to test it?

To accurately test this PR it's necessary to backup the TopDown Engine folder, located in `/client/Assets/ThirdParty` while on the main branch, then switch to the PR's branch and manually add the engine's folder in the corresponding folder of the client, after that you should verify no problems arise in Unity and that the game can be played as expected.

## Checklist
